src/stardict_lib.cpp

```diff
@@ -1032,9 +1067,25 @@ const gchar *Libs::poGetNextWord(const gchar *sWord, glong *iCurrent)
     size_t iCurrentLib = 0;
     const gchar *word;
 
+    glong nextWordIdx = 0;
+    std::set<glong> wordIndices;
+
     for (size_t iLib = 0; iLib < oLib.size(); ++iLib) {
-        if (sWord)
-            oLib[iLib]->Lookup(sWord, iCurrent[iLib]);
+        wordIndices.clear();
+        if (sWord) {
+            if (oLib[iLib]->Lookup(sWord, wordIndices, nextWordIdx)) {
+                // In order to progress over words which have multiple entries,
+                // pick the smallest index larger than iCurrent.
+                for (auto &wordIdx : wordIndices) {
+                    if (wordIdx > iCurrent[iLib]) {
+                        iCurrent[iLib] = wordIdx;
+                        break;
+                    }
+                }
+            } else {
+                iCurrent[iLib] = nextWordIdx;
+            }
+        }
```

Comment:
I believe this is the most reasonable change to make to `poGetNextWord` to make the whole thing work -- but I couldn't find any users of `poGetNextWord` and it seems this code comes from qstardict but is not actually used by anything. As such, I'm not sure how I can test whether this specific change is correct.

Reply:
If it is dead code then just remove it. At first there was idea to share stardict_lib, but it seems unrealistic.

Reply:
Remove just this function or all of the `po*` methods which aren't being used?

Reply:
I've removed all of the `po*` iterator functions.